pkg/config/config.go

```diff
@@ -32,7 +32,7 @@ import (
 const (
 	// NewReplicaImpl is true if we using new processor
 	// new owner should be also switched on after it implemented
-	NewReplicaImpl = false
+	NewReplicaImpl = true
```

Comment:
we keep the flag mean it is possible will switch back to old one ? if not could delete the flag, make it as the default behaviour ?

Reply:
i wil clean all related codes in another pr after this pr merge.